core/templates/components/rubrics-editor/rubrics-editor.directive.html

```diff
@@ -57,7 +57,7 @@
               Delete
             </button>
             <button type="button"
-                    class="btn btn-default"
```

Comment:
Just a minor thing, I think btn-default doesn't exist in bootstrap 4. Can you remove that and use one of the outline classes mentioned here: https://getbootstrap.com/docs/4.0/components/buttons/#outline-buttons

Reply:
Ping @Showtim3

Reply:
Done.